easybuild/easyconfigs/i/IQ-TREE/IQ-TREE-2.1.2-foss-2020b.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'CMakeMake'
+
+name = 'IQ-TREE'
+version = '2.1.2'
+
+# HTTPS is not working
+homepage = 'http://www.iqtree.org/'
+description = """Efficient phylogenomic software by maximum likelihood"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
```

Comment:
@sassy-crick We can/should drop this to `gompi`, see https://github.com/sassy-crick/easybuild-easyconfigs/pull/6